ZAPD/Main.cpp

```diff
@@ -75,7 +75,7 @@ void ErrorHandler(int sig)
 }
 #endif
 
-int main(int argc, char* argv[])
+int32_t main(int32_t argc, char* argv[])
```

Comment:
The signature of a C `main` function should be either
```c
int main();
```
or 
```c
int main(int, char*[]);
```
So, please, change it back.